db/migrate/20150416231106_add_source_to_webhook.rb

```diff
@@ -0,0 +1,12 @@
+class AddSourceToWebhook < ActiveRecord::Migration
+  def up
+    add_column :webhooks, :source, :string, null: false, default: 'any_ci'
+    change_column_default :webhooks, :source, nil
```

Comment:
This makes it not null, it only makes legacy webhooks into CI hooks. I definitely don't want null sources.